.github/pull_request_template.md

```diff
@@ -19,7 +19,9 @@ If this PR introduces a new feature, it is better to list some use cases here, a
 
 ## Checklist
 
-1. Pre-commit or other linting tools are used to fix the potential lint issues.
-2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
-3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
-4. The documentation has been modified accordingly, like docstring or example tutorials.
+- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
+- [ ] CLA has been signed and all committers have signed the CLA in this PR.
```

Comment:
Contributors will be asked to sign CLA after creating the first PR.
If we put it in the checklist, maybe fresh contributors cannot find where to sign it before creating the PR.